### ISSUE TITLE
Weekly Check-in for Code-In, #2

### ISSUE THREAD

**Issue by geekychaser** [publiclab/plots2#3906]
Hi everybody :smiley: !

If you're new here, welcome, and please try some of our [first-timers-only issues](https://code.publiclab.org/#r=all). This is a check-in for recent new contributors and mentors, and across the Code-In and Outreachy groups! @publiclab/mentors 

We're SO EXCITED
What is your plan for this week? 

I will be reviewing a couple of PRs, opening up a few issues on gci dashboard and will send some prs this week.

If you have not planned yet, just leave a Hello! so that we know that you are in sync with us :arrows_clockwise: and doing well!

## Google Code-In

We're in our third week (?) of Code-In and it's been a huge amount of energy. What went well or poorly? 

@jywarren indicated that students are having trouble installing plots. They can read https://github.com/publiclab/plots2/issues/3858#issuecomment-436432623

https://github.com/publiclab/plots2/issues/3840

## Bottlenecks

In this spirit, where can we direct more mentor energy? What's stuck? 

Also, what is going well? Let's celebrate! 

I for one have truly enjoyed seeing so much mutual support in all the issues. And great friendliness 😄 🎉 

The coming weeks will be absolutely full of code :computer: , tasks :spiral_notepad:, fun :tada: and excitement :smiley:!!

As always, if you're waiting for a review, or if you're stuck, please request help here OR leave a comment with @publiclab/mentors for some input. 🙌 

Have a GREAT week :tada:  !

Thank you !

PS - who among mentors would like to volunteer to post this next week? Please leave a comment below!

**Comment by SidharthBansal** [comment]
Hello everyone,
Outreachy applicants are all curious for the result. We all are reading through your proposals. So, please wait until results are out.

GCI students are doing a great job. They have completed 46 tasks and 13 tasks are in progress.
Well done candidates. :tada:

**Comment by jywarren** [comment]
🙌🏽♥️⚡

**Comment by jywarren** [comment]
And remember our support tasks if you are helping to keep our community well-supported, or looking to build your leadership skills a bit! 

https://github.com/publiclab/plots2/labels/support

**Comment by Radhikadua123** [comment]
Hello everyone,

This week I plan to work on merging my previous pull requests, replacing markdown parser and helping out with creating fto issues.

Thanks!

**Comment by faithngetich** [comment]
Hello everyone, 

My plan this week is to implement any feedback given on my pending [PR](https://github.com/publiclab/infragram-js/pull/9). Otherwise, I'll start working on a new task that's yet TBD. Suggestions are welcomed.

**Comment by jywarren** [comment]
I've been seeing some great leadership from folks helping review others' code, like @Paarmita, @CoderJolly, @cesswairimu, @shivani-kapania, @Radhikadua123 and many others! Great work folks.

**Comment by jywarren** [comment]
Did someone want to volunteer to do a check-in this coming week?

**Comment by CoderJolly** [comment]
How can we do so ?

**Comment by jywarren** [comment]
Hi, @CoderJolly -- so, you can copy and paste the content of this check-in into a new issue, and update the sections needed to keep it current. But don't worry, we can help you format it once you start it off. We'd love to have your help in checking in with folks around the community this week!

**Comment by jywarren** [comment]
I do want to include: shall we recommend that reviewers turn off most notifications in the "Watching" menu for plots2, since it can be overwhelming, and mainly listen to mentions for @publiclab/reviewers (and likewise for @publiclab/mentors?)

Second, I'm writing a brief report of our projects to post on PublicLab.org (like these! https://publiclab.org/tag/webwg) and would love to include any stats or info folks wanted to suggest!

**Comment by jywarren** [comment]
Yes, this is a problem! We haven't figured out a way around it yet for newcomers, but what we can do is add you as a collaborator and then you will be able to do it. I've done this for you just now and can for anyone else as well. Thanks for mentioning it!

**Comment by CodeSarthak** [comment]
Okay, thanks a lot 👍

**Comment by SidharthBansal** [comment]
Can anyone please open the checking for this week?

**Comment by SidharthBansal** [comment]
And close this issue?

**Comment by CoderJolly** [comment]
Hi @jywarren, I was away from keyboard lately bt now will do it

**Comment by CoderJolly** [comment]
> Hi, @CoderJolly -- so, you can copy and paste the content of this check-in into a new issue, and update the sections needed to keep it current. But don't worry, we can help you format it once you start it off. We'd love to have your help in checking in with folks around the community this week!

Hi @jywarren, I was away from keyboard lately bt now will do it

**Comment by CoderJolly** [comment]
> Hey, @jywarren , now that you have mentioned it, I am not able to reference @publiclab/reviewers. It does not become a tag and stays normal text. Help needed.

Is it with collaborators only or do the members also face this problem ?

**Comment by CodeSarthak** [comment]
Hey @jywarren I am still not able to reference @publiclab/reviewers.

**Comment by CodeSarthak** [comment]
One more thing,  I think we are running really low on tasks for GCI, only about 7-8 are left. @jywarren @SidharthBansal

**Comment by SidharthBansal** [comment]
I will try to publish some tomorrow. Can @publiclab/mentors drafts the
tasks so that I can publish them.
